Cython/Utility/ModuleSetupCode.c

```diff
@@ -7,6 +7,14 @@
   #define CYTHON_LIMITED_API 1
```

Comment:
Not sure what to do with this. It seems a decision that should really only be taken in one place. It's guarded by the `!defined()` check above, but that still means that modules generated by other Cython versions (older or newer) must define this flag if they have something to say in this regard. It looks safe but could still proof fragile in the future.